Jenkinsfile

```diff
@@ -0,0 +1,65 @@
+podTemplate(
+  cloud: 'openshift',
+  containers: [
+    containerTemplate(name: 'jnlp',
+      image: 'image-registry.openshift-image-registry.svc:5000/oasis/jenkins-agent-oasis:latest',
+      args: '${computer.jnlpmac} ${computer.name}',
+      envVars: [
+      secretEnvVar(key: 'OASIS_RHSM_USERNAME', secretName: 'oasis-rhsm', secretKey: 'username'),
```

Comment:
Indentation gets weird here. Are there changes between tabs and spaces?

Reply:
Probably!

Reply:
Turns out that's how it was actually indented, but the built-in vim indentation helper liked it this way. I made some tweaks.